app/src/main/java/topads/tokopedia/com/tinkersample/utils/UserPrefs.kt

```diff
@@ -0,0 +1,33 @@
+package topads.tokopedia.com.tinkersample.utils
+
+import android.content.Context
+import android.content.SharedPreferences
+
+/**
+ * Author errysuprayogi on 10,January,2020
+ */
+class UserPrefs private constructor(private val context: Context) {
+    private val settings: SharedPreferences
+    fun saveString(key: String?, value: String?) {
+        settings.edit().putString(key, value).commit()
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/errysuprayogi/TinkerSampleApp/blob/cac0a22a5fa97c10f7f94a3656ced67ae51876c9/app/src/main/java/topads/tokopedia/com/tinkersample/utils/UserPrefs.kt#L12"></span>Consider using <code>apply()</code> instead; <code>commit</code> writes its data to persistent storage immediately, whereas <code>apply</code> will handle it in the background</td>
    </tr>
  </tbody>
</table>